src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  pagesCount: number
+  onSelectedPage: (page: number) => void
+  selectedPage: number
+  firstItem: number
+  lastItem: number
+};
+
+export const Pagination: React.FC<Props> = ({
+  pagesCount,
+  onSelectedPage,
+  selectedPage,
+  firstItem,
+  lastItem,
+}) => {
+  const totalPages = [];
+  const itemsOnPage = [];
+
+  for (let i = 1; i <= pagesCount; i += 1) {
+    totalPages.push(i);
+  }
+
+  for (let i = firstItem; i <= lastItem; i += 1) {
+    itemsOnPage.push(i);
+  }
```

Comment:
redundant this loop
```suggestion
```
instead use items variable in App component and slice items from firstItem to lastItem and just pass in Pagination component sliced items.